src/slider/src/Slider.tsx

```diff
@@ -129,13 +127,22 @@ export default defineComponent({
       uncontrolledValueRef
     )
 
+    const wittyPlacementRef = computed(() => {
```

Comment:
```suggestion
    const mergedPlacementRef = computed(() => {
```

Follow the covention of the library.